Definitions/Key

```diff
@@ -1 +1,26 @@
-
+class Key:
+    def __init__(self, symbol, startNote, isMajor):
+        this.symbol = symbol
+        this.startNote = startNote%12
+        this.isMajor = isMajor
+    
+    def getScale(self):
```

Comment:
I think this is an elegant implementation of this logic, but I'm not sure it's wise to put it in an instance method. Since you call it in the `getScaleDegree` function below, it would be better to just store the output on an instance variable rather than having to recalculate each time. You could use this function almost as is (see below) and just call it within the `__init__` and store the value on the instance.

Also, since the ultimate goal here is to make a lookup table to see if a pitch exists within a key and then to determine what scale degree it is, it would be better to store this in a dictionary (lookup time of O(1)) rather than a list (lookup time of O(n)). Each key would be a number (cast to a string), and the value would also be a number, but would indicate the scale degree. As a brief example of G major:

```
{
    "7": 1, # 7 = pitch in midi value, 1 = scale degree 1,
    "9": 2,
    "11" : 3,
    ...
}
```